### PR TITLE
Versioning script

### PR DESCRIPTION
**Pull request by alvarolmedo**

### CODE REVIEW

File: flume/neore/rpm/get_version_string.sh [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+            echo 1
+        else
+            echo 0
+        fi
+    ;;
+    *)  echo 1 ;;
+   esac
+}
+
+#cd $1
```
Comment (fgalan) [inline flume/neore/rpm/get_version_string.sh]: Leftfover comment?

File: flume/neore/rpm/get_version_string.sh [hunk 2]
```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+#
+# Copyright 2013 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
```
Comment (frbattid) [inline flume/neore/rpm/get_version_string.sh]: Should be "This file is part of fiware-connectors (FI-WARE project)."

File: flume/neore/rpm/get_version_string.sh [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+#
+# Orion Context Broker is free software: you can redistribute it and/or
```
Comment (frbattid) [inline flume/neore/rpm/get_version_string.sh]: "fiware-connectors is free..."

File: flume/neore/rpm/get_version_string.sh [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
```
Comment (frbattid) [inline flume/neore/rpm/get_version_string.sh]: "fiware-connectors is distributed..."

File: flume/neore/rpm/get_version_string.sh [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
```
Comment (frbattid) [inline flume/neore/rpm/get_version_string.sh]: "... along with fiware-connectors".

File: flume/neore/rpm/get_version_string.sh (hunk 2 above)
Comment (fgalan) [inline flume/neore/rpm/get_version_string.sh]: Fixed in 97f56e68bb78d3b4711b8e95edc56f5d42c9494a

File: flume/neore/rpm/get_version_string.sh (hunk 3 above)
Comment (fgalan) [inline flume/neore/rpm/get_version_string.sh]: Fixed in 97f56e68bb78d3b4711b8e95edc56f5d42c9494a

File: flume/neore/rpm/get_version_string.sh (hunk 4 above)
Comment (fgalan) [inline flume/neore/rpm/get_version_string.sh]: Fixed in 97f56e68bb78d3b4711b8e95edc56f5d42c9494a

File: flume/neore/rpm/get_version_string.sh (hunk 5 above)
Comment (fgalan) [inline flume/neore/rpm/get_version_string.sh]: Fixed in 97f56e68bb78d3b4711b8e95edc56f5d42c9494a

File: flume/neore/rpm/get_version_string.sh (hunk 1 above)
Comment (fgalan) [inline flume/neore/rpm/get_version_string.sh]: NTC

**Comment by fgalan** [comment]
LGTM

**Comment by frbattid** [comment]
LGTM

**Pull request merged**